packages/dev-utils/src/env.ts

```diff
@@ -0,0 +1,19 @@
+import * as _ from 'lodash';
+import * as process from 'process';
+
+export const env = {
+    parseBoolean(key: string): boolean {
+        let isTrue: boolean;
+        const envVarvalue = process.env[key];
+        if (process.env.SOLIDITY_COVERAGE === 'true') {
+            isTrue = true;
+        } else if (process.env.SOLIDITY_COVERAGE === 'false' || _.isUndefined(process.env.SOLIDITY_COVERAGE)) {
```

Comment:
Erh, instead of hard-coding `SOLIDITY_COVERAGE` in these conditionals, shouldn't this be `envVarValue`?